src/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -75,19 +75,17 @@ bool ShootGoalTactic::isEnemyAboutToStealBall() const
     // Our rectangle class does not have the concept of rotation, so instead
     // we rotate all the robot positions about the origin so we can construct
     // a rectangle that is aligned with the axis
-    Angle theta = -robot->orientation();
+    Vector front_of_robot_dir =
+        Vector(robot->orientation().cos(), robot->orientation().sin());
 
-    Point rotated_baller_position = robot->position().rotate(theta);
-
-    Rectangle area_in_front_of_rotated_baller(
-        rotated_baller_position - Vector(0, 3 * ROBOT_MAX_RADIUS_METERS),
-        rotated_baller_position +
-            Vector(5 * ROBOT_MAX_RADIUS_METERS, 3 * ROBOT_MAX_RADIUS_METERS));
+    Rectangle baller_frontal_area = Rectangle(
+        (robot->position() + front_of_robot_dir.perp().norm(ROBOT_MAX_RADIUS_METERS)),
+        robot->position() + front_of_robot_dir.norm(3 * ROBOT_MAX_RADIUS_METERS) -
```

Comment:
Can we explain and/or parameterize the magic number 3?

Reply:
+1 - I'd probably parameterize `3 * ROBOT_MAX_RADIUS` so we're dealing with an absolute distance rather then a scaling factor, makes it easier to reason about.

Reply:
changed to a parameter with a description.